.github/workflows/ci-dgraph-ldbc-tests.yml

```diff
@@ -0,0 +1,73 @@
+name: ci-dgraph-ldbc-tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: "30 * * * *"
+jobs:
+  dgraph-ldbc-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Dgraph
+        uses: actions/checkout@v3
+      - name: Checkout LDBC query repo
+        uses: actions/checkout@v3
+        with:
+          repository: dgraph-io/ldbc
+          token: ${{ secrets.DEPLOY_KEYS_GH_DGRAPHIO_LDBC }}
+          ref: master
+          path: ldbc
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install protobuf-compiler
+        run: sudo apt-get install -y protobuf-compiler
+      - name: Check protobuf
+        run: |
+          cd ./protos
+          go mod tidy
+          make regenerate
+          git diff --exit-code -- .
+      - name: Make Docker Image
+        run: make image-local
```

Comment:
can replace with 
```
      - name: Make Linux Build and Docker Image
        run: make docker-image # this internally builds dgraph binary
```
which will build the binary and make the docker image (tagged with local)